sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/Speller.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package com.azure.search.documents.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import java.util.Collection;
+
+/**
+ * Defines values for Speller.
+ */
+public final class Speller extends ExpandableStringEnum<Speller> {
```

Comment:
Is this class the same as `QuerySpellerType`? Is this a Swagger transform issue happening where both are being generated?

Reply:
@alzimmermsft it does seem to be the same as `QuerySpellerType` but I couldn't find any swagger transform related to either. I'll add a transform to rename `Speller` to `QuerySpellerType`.